2024-07-18 05:11:38.517 WARNING fixture:conftest.py:60: Process started
2024-07-18 05:11:38.518 INFO fixture:conftest.py:61: Process ongoing..
2024-07-18 05:11:38.520 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.address`.
2024-07-18 05:11:38.521 DEBUG faker.factory:factory.py:95: Provider `faker.providers.address` has been localized to `en_US`.
2024-07-18 05:11:38.521 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.automotive`.
2024-07-18 05:11:38.522 DEBUG faker.factory:factory.py:95: Provider `faker.providers.automotive` has been localized to `en_US`.
2024-07-18 05:11:38.522 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.bank`.
2024-07-18 05:11:38.523 DEBUG faker.factory:factory.py:86: Specified locale `en_US` is not available for provider `faker.providers.bank`. Locale reset to `en_GB` for this provider.
2024-07-18 05:11:38.524 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.barcode`.
2024-07-18 05:11:38.524 DEBUG faker.factory:factory.py:95: Provider `faker.providers.barcode` has been localized to `en_US`.
2024-07-18 05:11:38.524 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.color`.
2024-07-18 05:11:38.525 DEBUG faker.factory:factory.py:95: Provider `faker.providers.color` has been localized to `en_US`.
2024-07-18 05:11:38.525 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.company`.
2024-07-18 05:11:38.527 DEBUG faker.factory:factory.py:95: Provider `faker.providers.company` has been localized to `en_US`.
2024-07-18 05:11:38.527 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.credit_card`.
2024-07-18 05:11:38.527 DEBUG faker.factory:factory.py:95: Provider `faker.providers.credit_card` has been localized to `en_US`.
2024-07-18 05:11:38.527 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.currency`.
2024-07-18 05:11:38.528 DEBUG faker.factory:factory.py:95: Provider `faker.providers.currency` has been localized to `en_US`.
2024-07-18 05:11:38.528 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.date_time`.
2024-07-18 05:11:38.530 DEBUG faker.factory:factory.py:95: Provider `faker.providers.date_time` has been localized to `en_US`.
2024-07-18 05:11:38.530 DEBUG faker.factory:factory.py:106: Provider `faker.providers.emoji` does not feature localization. Specified locale `en_US` is not utilized for this provider.
2024-07-18 05:11:38.530 DEBUG faker.factory:factory.py:106: Provider `faker.providers.file` does not feature localization. Specified locale `en_US` is not utilized for this provider.
2024-07-18 05:11:38.530 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.geo`.
2024-07-18 05:11:38.531 DEBUG faker.factory:factory.py:95: Provider `faker.providers.geo` has been localized to `en_US`.
2024-07-18 05:11:38.531 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.internet`.
2024-07-18 05:11:38.532 DEBUG faker.factory:factory.py:95: Provider `faker.providers.internet` has been localized to `en_US`.
2024-07-18 05:11:38.532 DEBUG faker.factory:factory.py:106: Provider `faker.providers.isbn` does not feature localization. Specified locale `en_US` is not utilized for this provider.
2024-07-18 05:11:38.532 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.job`.
2024-07-18 05:11:38.533 DEBUG faker.factory:factory.py:95: Provider `faker.providers.job` has been localized to `en_US`.
2024-07-18 05:11:38.533 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.lorem`.
2024-07-18 05:11:38.534 DEBUG faker.factory:factory.py:95: Provider `faker.providers.lorem` has been localized to `en_US`.
2024-07-18 05:11:38.534 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.misc`.
2024-07-18 05:11:38.534 DEBUG faker.factory:factory.py:95: Provider `faker.providers.misc` has been localized to `en_US`.
2024-07-18 05:11:38.534 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.passport`.
2024-07-18 05:11:38.534 DEBUG faker.factory:factory.py:95: Provider `faker.providers.passport` has been localized to `en_US`.
2024-07-18 05:11:38.534 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.person`.
2024-07-18 05:11:38.536 DEBUG faker.factory:factory.py:95: Provider `faker.providers.person` has been localized to `en_US`.
2024-07-18 05:11:38.536 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.phone_number`.
2024-07-18 05:11:38.537 DEBUG faker.factory:factory.py:95: Provider `faker.providers.phone_number` has been localized to `en_US`.
2024-07-18 05:11:38.538 DEBUG faker.factory:factory.py:106: Provider `faker.providers.profile` does not feature localization. Specified locale `en_US` is not utilized for this provider.
2024-07-18 05:11:38.538 DEBUG faker.factory:factory.py:106: Provider `faker.providers.python` does not feature localization. Specified locale `en_US` is not utilized for this provider.
2024-07-18 05:11:38.538 DEBUG faker.factory:factory.py:106: Provider `faker.providers.sbn` does not feature localization. Specified locale `en_US` is not utilized for this provider.
2024-07-18 05:11:38.538 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.ssn`.
2024-07-18 05:11:38.540 DEBUG faker.factory:factory.py:95: Provider `faker.providers.ssn` has been localized to `en_US`.
2024-07-18 05:11:38.541 DEBUG faker.factory:factory.py:106: Provider `faker.providers.user_agent` does not feature localization. Specified locale `en_US` is not utilized for this provider.
2024-07-18 05:11:38.542 DEBUG urllib3.connectionpool:connectionpool.py:1048: Starting new HTTPS connection (1): dev.agentum.pro:443
2024-07-18 05:11:38.898 DEBUG urllib3.connectionpool:connectionpool.py:546: https://dev.agentum.pro:443 "POST /backend/api/deal/create-with-client HTTP/1.1" 200 None
2024-07-18 05:11:38.898 CRITICAL root:helpers_api.py:27: ___response code- 200
2024-07-18 05:11:38.899 CRITICAL root:helpers_api.py:28: ___response body- {'deal_id': 5237}
2024-07-18 05:11:38.900 DEBUG tests_api.utils.logger:logger.py:23: > GET https://dev.agentum.pro/backend/api/deal/5237 {'method': 'GET', 'url': 'https://dev.agentum.pro/backend/api/deal/5237', 'params': None, 'headers': {'Authorization': 'Bearer YwQogs0aDUHzYIgAIN88OBaTDiKesDmAdciD9vkz'}}
2024-07-18 05:11:38.901 DEBUG urllib3.connectionpool:connectionpool.py:1048: Starting new HTTPS connection (1): dev.agentum.pro:443
2024-07-18 05:11:39.416 DEBUG urllib3.connectionpool:connectionpool.py:546: https://dev.agentum.pro:443 "GET /backend/api/deal/5237 HTTP/1.1" 200 None
2024-07-18 05:11:39.417 DEBUG tests_api.utils.logger:logger.py:27: < 200 https://dev.agentum.pro/backend/api/deal/5237 {"data":{"name":"<bound method Provider.name of <faker.providers.person.en_US.Provider object at 0x7f7ce00583d0>>","deal":{"id":5237,"type":{"value":"sell","label":"\u041f\u0440\u043e\u0434\u0430\u0436\u0430"},"status":{"value":"active","label":"\u0410\u043a\u0442\u0438\u0432\u043d\u0430\u044f"},"stages":{"current_stage":"sell_first_touch","all_stages":["sell_first_touch","sell_in_work","sell_showings","sell_deposit","sell_deal"]},"agent_reward":0,"comment":""},"sell":{"id":164129,"uuid":null,"created_at":"2024-07-18T05:11:38.000000Z","updated_at":"2024-07-18T05:11:38.000000Z","is_parsed":false,"action_type":"sell","object_type":null,"object_sub_type":[],"house_type":null,"price":null,"area":null,"land_area":null,"kitchen_area":null,"rooms":null,"floor":null,"floors":null,"address_raw":"","address_formatted":"","lat":null,"lon":null,"built_year":null,"walls_type":null,"decoration":null,"description":"","keys":null,"land_status":null,"renovation":null,"time_to_city":null,"secured":null,"is_apartments":null,"garage_type":null,"car_place_type":null,"additions":[],"communications":[],"furniture":[],"living_area":null,"is_communication_price_included":null,"is_subrent":null,"deposit":null,"zalog":null,"neighbours":null,"possible_tenant":null,"rental_period":[],"garage_facilities":[],"sell_purpose":null,"metro":[]},"rent_out":null,"purchase":null,"rent":null,"client":{"id":4026,"phone":"79905887434"},"presentation":null,"created_at":"2024-07-18T05:11:38.000000Z"}}
2024-07-18 05:11:39.417 DEBUG tests_api.utils.logger:logger.py:35: curl -X GET -H 'Accept: */*' -H 'Accept-Encoding: gzip, deflate' -H 'Authorization: Bearer YwQogs0aDUHzYIgAIN88OBaTDiKesDmAdciD9vkz' -H 'Connection: keep-alive' -H 'User-Agent: python-requests/2.31.0' https://dev.agentum.pro/backend/api/deal/5237
2024-07-18 05:11:39.420 INFO fixture:conftest.py:63: Process ended